test_manager/features/steps/tck_step_implementations.py

```diff
@@ -50,6 +43,17 @@
 
 from dispatcher.dispatcher import Dispatcher
 
+def get_absolute_path(filepath_from_root_repo: str) -> str:
+    return os.path.abspath(os.path.dirname(os.getcwd()) + "/" + filepath_from_root_repo)
+
+def get_docker_host_ip():
+    cmd = "ip -4 addr show docker0 | grep -Po 'inet \\K[\\d.]+'"
```

Comment:
@agosh01 Isn't the command here, specific to linux environment?

Reply:
yes, window will be likely virtual machine with docket which command will look like "docker-machine ip default". 

Can you provide more info about ask ? Do you have windows usecase for this ? 

I can add check for windows or linux and do appropriate execution

Reply:
Does current changes support running of docker container on linux and windows. Or only linux?

Reply:
only linux but we do couple thing add check to restrict the user to run this only on linux or add windows compatibility if there is usecase for this